### PR TITLE
PCLOUDS-4015 - service logs schemas

### PR DESCRIPTION
**Pull request by nikita-paskal-dynatrace**

### CODE REVIEW

File: src/config_logs/google-data-transfer/googlge-data-transfer.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (szymon-nagel-dt) [inline src/config_logs/google-data-transfer/googlge-data-transfer.json:1]: Why you didn't add a storage_transfer_job entity log linking despite avilable logs in GPC?
image

**Review by szymon-nagel-dt — CHANGES_REQUESTED** [review]

File: src/config_logs/google-hybrid-connectivity/google-hybrid-connectivity.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (nikita-paskal-dynatrace) [inline src/config_logs/google-hybrid-connectivity/google-hybrid-connectivity.json:1]: won't do, nothing to add

Comment (szymon-nagel-dt) [reply src/config_logs/google-hybrid-connectivity/google-hybrid-connectivity.json:1]: Can delete this empty file then

File: src/config_logs/google-datastore/google-datastore-database.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
```
Comment (szymon-nagel-dt) [inline src/config_logs/google-datastore/google-datastore-database.json:1]: Again , the same as above but leave as it is

File: src/config_logs/google-datastore/google-datastore-index.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
+{
```
Comment (szymon-nagel-dt) [inline src/config_logs/google-datastore/google-datastore-index.json:1]: I think this can be deleted

**Review by szymon-nagel-dt — APPROVED** [review]
Besides changes mentioned Looks good to me :)

**Review by szymon-nagel-dt — APPROVED** [review]

**Pull request merged**